databroker/headersource/base.py

```diff
@@ -1,8 +1,16 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 from . import core
+import numpy as np
 
 
+def _sanitize_np(val):
```

Comment:
There is probably a longer discussion to be had about where this should be done (would be better to push lower for ex the hdf5 backend?